pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,42 @@
+{ 
+  pkgs ? import <nixpkgs> {}
+  , lib ? pkgs.lib
+  , stdenv ? pkgs.stdenv
+  , makeWrapper ? pkgs.makeWrapper
+  , fetchFromGitHub ? pkgs.fetchFromGitHub
+  , fakeSha256 ? pkgs.lib.fakeSha256
+  , cmake ? pkgs.cmake
+  , pkg-config ? pkgs.pkg-config
+  , cryptopp ? pkgs.cryptopp
+  , qtbase ? pkgs.qt5.qtbase
+}:
+
+stdenv.mkDerivation {
+
+  name = "OTPGen";
+
+  meta = {
+    description = "MULTI-PURPOSE OTP TOKEN GENERATOR WRITTEN IN C++ AND QT WITH ENCRYPTION AND SYSTEM TRAY ICON. SUPPORTS TOTP, HOTP, AUTHY AND STEAM AND CUSTOM DIGIT LENGTHS AND PERIODS.";
+    license = lib.licenses.mit;
+    homepage = "https://github.com/magiruuvelvet/OTPGen";
+    maintainers = [ "Scott Hamilton <sgn'dot'hamilton'at'protonmail'dot'com>" ];
+    priority = "10";
+    timeout = 3600;
+    platforms = stdenv.lib.platforms.linux;
+  };
+
+  nativeBuildInputs = [ cmake  ];
+  buildInputs = [ cryptopp qtbase ];
+  depsBuildBuild = [ pkg-config  ];
+
+  src = fetchFromGitHub {
+    owner = "magiruuvelvet";
+    repo = "OTPGen";
+    rev = "97c7aabfe618c174da5b2ab3bf62e828e3d1798f";
+    sha256 = "sha256:0nlhjkmr5a8ivpbk1hys6zkrbrbs1kghsimzfw9z59ymnk9k10ll";
```

Comment:
you can skip the "sha256:" prefix

Reply:
fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/5664fd4112b3dfedc09e489d46a269f97d566a61